scripts/setup/vscode.sh

```
@@ -0,0 +1,139 @@
+        return 0 # インストールされていなければエラーではない
+    fi
+    log_installed "Visual Studio Code"
+
+    # ターゲットディレクトリの作成
+    mkdir -p "$vscode_target_dir"
+
+    # 設定ファイルのシンボリックリンクを作成
+    local linked_count=0
+    for file in "$config_dir"/*; do
+        if [ -f "$file" ]; then
+            local filename=$(basename "$file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)